packages/twenty-front/src/modules/settings/accounts/components/ImapConnectionForm.tsx

```diff
@@ -0,0 +1,119 @@
+import { Control, Controller } from 'react-hook-form';
+
+import { Select } from '@/ui/input/components/Select';
+import { TextInput } from '@/ui/input/components/TextInput';
+import styled from '@emotion/styled';
+import { useLingui } from '@lingui/react/macro';
+import { H2Title } from 'twenty-ui/display';
+import { Section } from 'twenty-ui/layout';
+import { ImapConnectionParams } from '~/generated/graphql';
+
+const StyledFormContainer = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: ${({ theme }) => theme.spacing(4)};
+`;
+
+type ImapConnectionFormProps = {
+  control: Control<ImapConnectionParams>;
+  isEditing: boolean;
+  defaultValues?: Partial<ImapConnectionParams>;
+};
+
+export const ImapConnectionForm = ({
```

Comment:
can you prefix this component like the other ones from the same folder ? and rename the file as well ?